server/routes/index.js

```diff
@@ -45,7 +45,11 @@ router.post("/testHook", cors(), async (req, res) => {
         const hookResponse = await axios.post(hook);
         res.sendStatus(hookResponse.status);
     } catch (err) {
-        log.error(err);
+        log.error({
+            message: err.message,
+            statusCode: 404,
```

Comment:
@cfoskin do you think it makes sense to use `err.response.status` instead of 404 here?
Reason: for example testing of `https://www.google.com` as a hook returns `"message":"Request failed with status code 405","statusCode":404`

Reply:
yep it does